addons/block_code/ui/picker/categories/category_factory.gd

```diff
@@ -462,6 +467,33 @@ static func get_general_blocks() -> Array[Block]:
 	b.tooltip_text = "Play the audio stream with volume and pitch"
 	b.category = "Sounds"
 	block_list.append(b)
+#endregion
+#region Graphics
+
+	var viewport_width = ProjectSettings.get_setting("display/window/size/viewport_width")
+	var viewport_height = ProjectSettings.get_setting("display/window/size/viewport_height")
+
+	b = BLOCKS["parameter_block"].instantiate()
+	b.variant_type = TYPE_INT
+	b.block_format = "viewport width"
+	b.statement = "%s" % viewport_width
+	b.category = "Graphics | Viewport"
+	block_list.append(b)
+
+	b = BLOCKS["parameter_block"].instantiate()
+	b.variant_type = TYPE_INT
+	b.block_format = "viewport height"
+	b.statement = "%s" % viewport_height
```

Comment:
Since the viewport can change size, should we use this as a runtime replacement instead?
```suggestion
	b.statement = "get_viewport().get_visible_rect().size.y"
```